packages/manageState/CHANGELOG.md

```diff
@@ -0,0 +1,82 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [0.1.3]
+### Changed
+- No Changes
```

Comment:
Changelogs are per-package, and should not contain changes for other packages in a monorepo. You should remove all versions except `0.1.3` from this changelog and update this to:
```suggestion
## [0.1.3]
### Added
- Initial release
```